language/libra-vm/src/libra_transaction_executor.rs

```diff
@@ -307,23 +307,26 @@ impl LibraVM {
         &self,
         remote_cache: &StateViewCache<'_>,
         writeset_payload: &WriteSetPayload,
+        txn_data: Option<&TransactionMetadata>,
```

Comment:
Could this be `sender:  Option<AccountAddress>` (since that's the only thing we read from `txn_data`)?